src/megatron/energon/av/av_decoder.py

```diff
@@ -25,6 +25,13 @@
 from megatron.energon.av.av_data import AVData
 
 
+def initialize_av_container(input_container: av.container.InputContainer) -> None:
```

Comment:
Add documentation, why is this needed.
Cause 1:
Threading causes a bug which leads to a deadlock in ffmpeg when deallocating the object.
Cause 2:
Threading causes forking to break (what happens?)